grammar/argextend.mlp

```diff
@@ -46,17 +46,17 @@ let make_act loc act pil =
   make (List.rev pil)
 
 let make_prod_item = function
-  | ExtTerminal s -> <:expr< Grammar_API.Extend.Atoken (Grammar_API.CLexer.terminal $mlexpr_of_string s$) >>
+  | ExtTerminal s -> <:expr< API.Extend.Atoken (API.CLexer.terminal $mlexpr_of_string s$) >>
```

Comment:
IMO `API` should not be inserted here, similarly for the many occurrences along the patch.